workshops/vue/full-day/ch5.md

```diff
@@ -303,7 +303,7 @@ Edit the form to bind the `valid` property:
 Let's also disable our `Submit` button when form is not valid.
 
```

Comment:
since the `nameRules` property is added in the workshop _after_ this button is created, let's add a small note here to let the student know that `nameRules` will be added in the next step.

Reply:
Done :), thank you!